lib/field-locale.ts

```diff
@@ -1,15 +1,16 @@
 import { Channel } from './channel'
 import { MemoizedSignal } from './signal'
-import { FieldAPI, FieldInfo, Items, SerializedJSONValue } from './types'
+import { FieldAPI, FieldInfo, SerializedJSONValue } from './types'
+import { FieldLocaleType } from './types/utils'
 import { ValidationError } from './types/validation-error'
 
 export default class FieldLocale implements FieldAPI {
   id: string
   locale: string
-  type: string
+  type: FieldLocaleType
```

Comment:
Naming could be better. This is a field type, right? And no need to use the word 'type' in the name of type.